wbooks/src/app/screens/booklist/index.js

```diff
@@ -1,15 +1,32 @@
 import React, { Component } from 'react';
-import { View, StyleSheet } from 'react-native';
+import { 
+	SafeAreaView, 
+	View, 
+	StyleSheet, 
+	FlatList 
+} from 'react-native';
 import BookItem from './components/BookItem';
 import styles from './styles';
+import books from './books';
 
 class BookList extends Component {
-
+	 
 	render() {
+
 		return (
-			<View style={styles.container}>
-        <BookItem />
-			</View>
+			<SafeAreaView style={styles.container}>
+				<FlatList
+					data={books}
+					renderItem={({ item }) => 
```

Comment:
since we talked about arrow functions, create a function inside the class that manage this render. Let's call it `renderItem`

Reply:
done!

Reply:
done